packages/components/src/modal/style.scss

```diff
@@ -48,18 +40,18 @@
 
 @keyframes components-modal__appear-animation {
 	from {
-		margin-top: $grid-unit-40;
+		transform: translateY($grid-unit-40);
 	}
 	to {
-		margin-top: 0;
+		transform: translateY(0);
 	}
 }
 
 // Fix header to the top so it is always there to provide context to the modal
 // if the content needs to be scrolled (for example, on the keyboard shortcuts
 // modal screen).
 .components-modal__header {
-	box-sizing: border-box;
```

Comment:
Is this removed because it wasn't needed or for some other reason?

Reply:
Thanks for asking about this as I'd meant to double-check it. It didn't seem needed as this will apply the same rule: https://github.com/WordPress/gutenberg/blob/f10179b9aa025aaf3513871c051998ab89abc993/packages/edit-post/src/style.scss#L69-L72

But that could go wrong when `Modal` is used outside of WordPress packages like edit-post or edit-site. I've removed the `width` and set the `right:0` to avoid that potential issue. Though, it makes me wonder why the reset to `.components-modal__frame' is applied in both those packages. Perhaps we should just apply it to the component itself?

Reply:
The reason I ask is that I think we should be using `border-box` very liberally, as in my experience it helps simplify all the adjacent CSS. If the effect is the same, fine to remove of course.

Reply:
That makes sense. I just can't help wanting to remove the redundancy. Since I've triple-checked this I realized that the modal header height may end up one pixel taller than intended if `Modal` is used standalone (outside of the edit-* packages that apply the reset to the modal). So I've put it back 😅 .